src/Groups/matrices/stuff_field_gen.jl

```diff
@@ -0,0 +1,52 @@
+import Hecke: evaluate, field_extension, FinField, FinFieldElem, PolyElem
+
+export primitive_element
+
+function _change_type(f::PolyElem{T}) where T <: FinFieldElem
```

Comment:
Could you please add a comment (or even docstring) explaining what this does?

It seems to convert polynomials to fq_nmod_poly or fq_poly -- but what are the input polynomials you expect to deal with?

Reply:
Input polynomials are polynomials where the base ring is not `FqNmodFiniteField` (for example, `Nemo.GaloisField`). The function `_centralizer` works only for polynomials of type `fq_nmod_poly` or `fq_poly`.

Reply:
@fieker do we have another way to convert such polynomials?